### ISSUE TITLE
odd chromium error

### ISSUE THREAD

**Issue by EmilyTheDruid** [lakshyabatman/Bulk-uploader-NFT#8]
Hello there, I attempted to use this to of course, upload a bunch of NFT's to opensea, but for some reason the metamask plugin isn't being recognized by opensea. I log into metamask, navigate to open sea, hit the metamask option, and then it takes me to the metamask download page unaware I already have the metamask add on installed. Anybody else run into this issue?

**Comment by guardaserver** [comment]
Yes, I had that same issue, accounts needed updating, I contacted their support and submitted a ticket. Checked back after sometime and the problem was fixed https://form.jotform.com/220245836463052

**Comment by lakshyabatman** [comment]
Have you made sure metamask is there in your automation web browser?
